static/panes/editor.js

```diff
@@ -104,6 +105,11 @@ function Editor(hub, state, container) {
     });
     this.editor.getModel().setEOL(monaco.editor.EndOfLineSequence.LF);
 
+    // console.log(" Vim enabled? " , this.settings.useVim)
+
+    if ( this.settings.useVim )
+        monacoVim.initVimMode(this.editor, document.getElementById('vim-status'));
```

Comment:
In other places we're using `$('#vim-status')` to access things in the dom: would you mind changing just for consistency? Sorry to be a pain!